routes/index.js

```diff
@@ -1,9 +1,8 @@
 var express = require('express');
 var router = express.Router();
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
-});
+let indexController = require('../controllers');
 
+router.get('/', indexController.home);
+router.get('/userRegistration', indexController.userRegistration);
```

Comment:
```suggestion
```
1. Removed unneeded home page as it will be supplied by an html page
2. The userRegistration is a duplicate